<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item alignment="Qt::AlignLeft">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f1f1f1;
border: none;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>50</number>
       </property>
       <property name="topMargin">
        <number>100</number>
       </property>
       <property name="rightMargin">
        <number>50</number>
       </property>
       <property name="bottomMargin">
        <number>100</number>
       </property>
       <item>
        <widget class="QWidget" name="verticalWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>100</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 0px;; </string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>nda-logo.png</normaloff>nda-logo.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="styleSheet">
             <string notr="true">font-weight:500</string>
            </property>
            <property name="text">
             <string>Nigerian Defence Academy</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QWidget" name="verticalWidget_2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QPushButton" name="StartAClass">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #ffffff;
border: 1px solid #128531;
border-radius: 15px;
font-weight: 400;
height: 30px;
width: 100px;</string>
            </property>
            <property name="text">
             <string>Start A Class</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="EnrollAStudent">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #ffffff;
border: 1px solid #128531;
border-radius: 15px;
font-weight: 400;
height: 30px;
width: 100px;</string>
            </property>
            <property name="text">
             <string>Enroll A Student</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="CreateAClass">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #ffffff;
border: 1px solid #128531;
border-radius: 15px;
font-weight: 400;
height: 30px;
width: 100px;</string>
            </property>
            <property name="text">
             <string>Create A Class</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabWidget_2Page1">
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>484</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 18px</string>
          </property>
          <property name="text">
           <string>Start A Class</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="styleSheet">
           <string notr="true">border: none</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: white;
border:none;</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QWidget" name="widget_4" native="true">
          <property name="styleSheet">
           <string notr="true">background-color: white</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: white;</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="bottomMargin">
               <number>26</number>
              </property>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="styleSheet">
                 <string notr="true">font-size: 18px;</string>
                </property>
                <property name="text">
                 <string>Create A Class</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">font-size:15px;margin-bottom:10px;margin-top:10px;</string>
             </property>
             <property name="text">
              <string>Course Code</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="CourseCodeTextField">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:white;border-radius:10; border: 1px solid #000000;font-size:15px;</string>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:15px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">font-size:15px;margin-bottom:10px;margin-top:10px;</string>
             </property>
             <property name="text">
              <string>Course Title</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="CourseTitleTextField">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:white;border-radius:10; border: 1px solid #000000;font-size:15px;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="CreateClass">
             <property name="styleSheet">
              <string notr="true">background-color: #128531;color:white;font-weight:500;height:40px;margin-top:20px;border-radius:10px;font-size:15px;</string>
             </property>
             <property name="text">
              <string>Create Class</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: white;
border:none;</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>18</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>484</width>
               <height>32</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-size: 18px</string>
             </property>
             <property name="text">
              <string>My Classes</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_6" native="true">
          <property name="styleSheet">
           <string notr="true">background-color: white;</string>
          </property>
          <widget class="QPushButton" name="pushButton_7">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>111</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: white;
border: 1px solid #128531;
border-radius: 7px; font-weight: 500;</string>
           </property>
           <property name="text">
            <string>CSS 700</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QWidget" name="widget_7" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: white;</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="styleSheet">
              <string notr="true">font-size: 18px;</string>
             </property>
             <property name="text">
              <string>Enrol A Student</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_8" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QWidget" name="widget_9" native="true">
             <property name="maximumSize">
              <size>
               <width>500</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #F1F1F1;border-radius:10px;</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QWidget" name="widget_10" native="true">
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>383</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: #f1f1f1</string>
                </property>
                <widget class="QTextEdit" name="StudentIdTextField">
                 <property name="geometry">
                  <rect>
                   <x>20</x>
                   <y>60</y>
                   <width>256</width>
                   <height>50</height>
                  </rect>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color:white;border-radius:10; border: 1px solid grey;</string>
                 </property>
                 <property name="html">
                  <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="EnrollFaceId">
                 <property name="geometry">
                  <rect>
                   <x>20</x>
                   <y>290</y>
                   <width>256</width>
                   <height>51</height>
                  </rect>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: #000000;color:white;font-weight:500;height:40px;margin-top:20px;border-radius:10px;font-size:15px;</string>
                 </property>
                 <property name="text">
                  <string>Enroll Face ID</string>
                 </property>
                </widget>
                <widget class="QTextEdit" name="StudentNameTextField">
                 <property name="geometry">
                  <rect>
                   <x>20</x>
                   <y>160</y>
                   <width>256</width>
                   <height>50</height>
                  </rect>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color:white;border-radius:10; border: 1px solid grey;</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_7">
                 <property name="geometry">
                  <rect>
                   <x>20</x>
                   <y>116</y>
                   <width>256</width>
                   <height>38</height>
                  </rect>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font-size:15px;margin-bottom:10px;margin-top:10px;</string>
                 </property>
                 <property name="text">
                  <string>FULL NAME</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_8">
                 <property name="geometry">
                  <rect>
                   <x>20</x>
                   <y>16</y>
                   <width>256</width>
                   <height>38</height>
                  </rect>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font-size:15px;margin-bottom:10px;margin-top:10px;</string>
                 </property>
                 <property name="text">
                  <string>NDA NUMBER</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="AddStudent">
                 <property name="geometry">
                  <rect>
                   <x>20</x>
                   <y>230</y>
                   <width>256</width>
                   <height>51</height>
                  </rect>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: #128531;color:white;font-weight:500;height:40px;margin-top:20px;border-radius:10px;font-size:15px;</string>
                 </property>
                 <property name="text">
                  <string>Add Student</string>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QWidget" name="widget_11" native="true">
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: white</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QPushButton" name="pushButton_10">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;background-color:black;border-radius:5px;height:25px;</string>
             </property>
             <property name="text">
              <string>Back</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_12" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #ffffff</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="styleSheet">
                 <string notr="true">font-size: 18px;font-weight:bold;</string>
                </property>
                <property name="text">
                 <string>CSS 719</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color:green; color:white;border:none;border-radius:5px;</string>
                </property>
                <item>
                 <property name="text">
                  <string>10th March 2020</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_13" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #f1f1f1;</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_10">
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font-size:18px;</string>
                </property>
                <property name="text">
                 <string>Steganography</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item alignment="Qt::AlignTop">
            <widget class="QWidget" name="widget_14" native="true">
             <property name="styleSheet">
              <string notr="true">background-color:white;</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QTableWidget" name="tableWidget">
                <column>
                 <property name="text">
                  <string>Name</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Student ID</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
